src/wp-content/blogposts.njk

```diff
@@ -0,0 +1,33 @@
+---
+layout: base.njk
+permalink: single.php
+config:
+  asset_path: "<?= get_template_directory_uri() ?>"
+---
+
+{% block content %}
+  <?php while (have_posts()) : the_post(); ?>
+  <main aria-label="Content">
+    <h1 class="Post-title"><?= the_title() ?></h1>
+    <div class="Blog-wrapper">
+      <div class="Blog">
+        <div class="Content-wrapper">
+          <?= the_content() ?>
+        </div>
+      </div>
+      <aside class="Sidebar">
+        <dl class="Meta">
+          <dt class="published">Updated</dt>
+          <dd class="published"><?= the_time() ?></dd>
+          <dt>Author</dt>
+          <dd>
+            <ul class="Authorlist">
+              <li class="Author"><?= the_author() ?></li>
```

Comment:
Not sure why but the author name seems to be displayed twice in the preview.

Reply:
I looked at the user, in-case it was related to not having a first and last name but that made no difference.

Reply:
Looks like you might want https://developer.wordpress.org/reference/functions/get_the_author/ instead. `the_author()` echos and returns the author by default.

Reply:
ah, indeed. I didn't pay attention to that, I inferred the function's name based on the others.

Reply:
fixed